app/handlers/tests/test_handlers_resource.py

```diff
@@ -44,13 +44,42 @@ def test_create(self, resource_factory, kopf_info_mock, mock_api_client):
             "ts": ANY,
         }
 
+        mock_api_client.resource_update.assert_not_called()
         mock_api_client.resource_create.assert_called_once_with(resource_spec)
         logger_mock.info.assert_called_once_with("Got a create request: %s", spec)
         kopf_info_mock.assert_called_once_with(
             "", reason="Success", message=f"Created on Twingate as {resource.id}"
         )
         assert patch_mock.spec == {"id": resource.id}
 
+    def test_resource_import(self, resource_factory, kopf_info_mock, mock_api_client):
```

Comment:
nit: I would call this `test_update_resource_when_id_is_specified`. I would understand "import" if I'm aware of https://github.com/Twingate/kubernetes-operator/issues/245